.azuredevops/build-ci.yml

```
@@ -12,49 +12,77 @@ pr:
 variables:
   skipComponentGovernanceDetection: true
 
-pool: 'Lite'
+resources:
+  repositories:
+    - repository: m365Pipelines
+      type: git
+      name: 1ESPipelineTemplates/M365GPT
+      ref: refs/tags/release
+extends:
+  template: v1/M365.Official.PipelineTemplate.yml@m365Pipelines
+  parameters:
+    pool:
+      name: NewLarge-linux-1ES
```

I think the fact that this pipeline is trying to use a pool that doesn't exist today points to how little activity we see/expect on this repo. The pipeline is even disabled in the internal project. And the name of that non-existent pipeline suggests to me this doesn't need the Large pool.
```suggestion
      name: Small-1ES
```